main/musl/APKBUILD

```diff
@@ -12,7 +12,7 @@ depends_dev="!uclibc-dev"
 makedepends="$depends_dev"
 subpackages="$pkgname-dev $pkgname-dbg libc6-compat:compat:noarch"
 case "$BOOTSTRAP" in
-nocc)	pkgname="musl-dev"; subpackages="";;
+nocc)	pkgname="musl-dev"; subpackages=""; arch="noarch";;
```

Comment:
The musl package has headers which are architecture specific, this should not be done.